services/core-web/src/components/mine/MineComments.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
+import { getMineComments } from "@common/selectors/mineSelectors";
+import {
+  deleteMineComment,
+  createMineComment,
+  fetchMineComments,
+} from "@common/actionCreators/mineActionCreator";
+import CommentPanel from "@/components/common/comments/CommentPanel";
+
+const propTypes = {
+  mineGuid: PropTypes.string.isRequired,
+  comments: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)),
+  deleteMineComment: PropTypes.func.isRequired,
+  createMineComment: PropTypes.func.isRequired,
+  fetchMineComments: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  comments: [],
+};
+
+export class MineComments extends Component {
+  state = { loading: true };
+
+  componentDidMount() {
+    this.fetchComments();
+  }
+
+  handleRemoveComment = (commentGuid) => {
+    this.props.deleteMineComment(this.props.mineGuid, commentGuid).then(() => this.fetchComments());
+  };
+
+  handleAddComment = async (values) => {
+    const formValues = {
+      mine_comment: values.comment,
+    };
+    return this.props.createMineComment(this.props.mineGuid, formValues).then(() => {
+      this.fetchComments();
+    });
+  };
+
+  fetchComments() {
+    this.setState({ loading: true });
+    this.props.fetchMineComments(this.props.mineGuid).then(() => this.setState({ loading: false }));
+  }
+
+  render() {
+    return (
+      <div>
+        <CommentPanel
+          renderAdd
```

Comment:
renderAdd?

Reply:
`CommentPanel` can be a view-only component, the `renderAdd` prop allows users to add comments.

Reply:
Maybe "isViewOnly" would be a more clear name?

Reply:
Updated it 'RenderEditor', this was a pre-exisiting pattern and 'isViewOnly' would be the reverse logic

Reply:
I think you mis-referenced the new prop name when you changed it (renderEditor and isEditable)? I think isEditable sounds the most clear.

Reply:
ah, I forgot to save the file 🤦 
I originally changed it to 'isEditable' but that didn't accurately represent what was happening, so updated the name to 'renderEditor'

Reply:
I think we should try and stick to a common prefix convention for naming boolean variables in our JavaScript code. Because it's a weakly/untyped language, it makes it more clear especially at a glance what type the variable is intended to be. So starting with "is", "has", etc. I'm not going to be a huge stickler about it though. :P "renderEditor" doesn't sound as clear as "isEditable", "isEditMode", etc. in my opinion.